pkgs/shells/zsh/nix-zsh-completions/default.nix

```diff
@@ -14,16 +14,26 @@ stdenv.mkDerivation rec {
     sha256 = "0m8b9xgbz2nvk1q7m0gqy83gbqa49n062gymhk9x93zhbdh8vwky";
   };
 
-  installPhase = ''
-    mkdir -p $out/share/zsh/site-functions
-    cp _* $out/share/zsh/site-functions
+  installPhase = let
+    pluginDir = if ohMyZsh then "plugins/nix" else "share/zsh/site-functions";
+    compDir   = if ohMyZsh then "completions" else "share/zsh/site-functions";
```

Comment:
Is this way of packaging ohMyZsh plugins documented somewhere? I can't find anything else in nixpkgs using this, and it feels like that shouldn't be part of the package declaration, but rather of the code that wants to use the package (by wrapping the paths how ohmyzsh expects them or so).